packages/replay/src/types.ts

```diff
@@ -1,5 +1,11 @@
+import { GLOBAL_OBJ } from '@sentry/utils';
 import type { eventWithTime, recordOptions } from 'rrweb/typings/types';
 
+// exporting WINDOW from within this package instead of importing it from @sentry/browser
+// this avoids the Browser package being bundled into the CDN bundle as well as a
+// circular dependency between the Browser and Replay packages in the future
+export const WINDOW = GLOBAL_OBJ as typeof GLOBAL_OBJ & Window;
```

Comment:
Hmm, I think `types.ts` is maybe not the ideal place to put this? As I would expect to have only type exports from here, I guess?

I would suggest the following:

1. move `sessions/contants.ts` to top level next to `types.ts` (IMHO makes more sense anyhow, as we also import from this from the main index.ts)
2. Put WINDOW in there

What do you think?

Reply:
Oh, good point. Will move it. Thanks!

Reply:
[ddf3ca4](https://github.com/getsentry/sentry-javascript/pull/6386/commits/ddf3ca4a421b389275e66dde269248cc8f6ab5bd)